spring-batch/src/main/java/org/baeldung/conditionalflow/ConditionalFlowApplication.java

```diff
@@ -0,0 +1,18 @@
+package org.baeldung.conditionalflow;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ConditionalFlowApplication implements CommandLineRunner {
+
+    public static void main(String[] args) {
+        SpringApplication.run(ConditionalFlowApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        System.out.println("Running and exiting");
```

Comment:
Could we prefer a proper logger instead of System.x - we already have a logback.xml in the module.

Reply:
Done